lib/core/FileHandlerAsyncTask.js

```diff
@@ -9,7 +9,7 @@ var FileHandleGenerator = function(fileName, truncate) {
 var _execute = function(callback) {
 	this.callback = callback;
 	var self = this;
-	var mode = 'a+';
+	var mode = 'r+';
```

Comment:
I didn't dig too far into that part of the code, but I thought that was taken care of by self.truncate and the alternate if clause that uses w+.